Storage/Clients.swift

```diff
@@ -3,7 +3,6 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import Shared
-import SwiftyJSON
```

Comment:
Well that was a quick PR turns out 😆 Thank you!

Reply:
yes, it sure was :)